packages/core/admin/admin/src/components/Roles/RoleList/RoleRow.js

```diff
@@ -1,48 +1,61 @@
-import React from 'react';
+import { Box, Row, Td, Text, Tr, IconButton, BaseCheckbox } from '@strapi/parts';
 import PropTypes from 'prop-types';
-import { CustomRow } from '@buffetjs/styles';
-import { IconLinks, Text } from '@buffetjs/core';
+import React from 'react';
 import { useIntl } from 'react-intl';
-
 import RoleDescription from './RoleDescription';
 
-const RoleRow = ({ role, onClick, links, prefix }) => {
+const RoleRow = ({ onToggle, id, name, description, usersCount, isChecked, icons }) => {
   const { formatMessage } = useIntl();
-  const number = role.usersCount;
-  const text = formatMessage(
-    { id: `Roles.RoleRow.user-count.${number > 1 ? 'plural' : 'singular'}` },
-    { number }
+
+  const usersCountText = formatMessage(
+    { id: `Roles.RoleRow.user-count.${usersCount > 1 ? 'plural' : 'singular'}` },
+    { number: usersCount }
   );
 
   return (
-    <CustomRow onClick={onClick}>
-      {prefix && <td style={{ width: 55 }}>{prefix}</td>}
-      <td>
-        <Text fontWeight="semiBold">{role.name}</Text>
-      </td>
-      <td>
-        <RoleDescription>{role.description}</RoleDescription>
-      </td>
-      <td>
-        <Text>{text}</Text>
-      </td>
-      <td>
-        <IconLinks links={links} />
-      </td>
-    </CustomRow>
+    <Tr>
+      {Boolean(onToggle) && (
+        <Td>
+          <BaseCheckbox name="role-checkbox" onValueChange={() => onToggle(id)} value={isChecked}  aria-label={`Select ${name} for bulk actions`} />
```

Comment:
I think the label should be internationalized, (sorry for not having put this one on the previous review 😬 )